modules/javafx.controls/src/test/java/test/javafx/scene/control/ListViewKeyInputTest.java

```diff
@@ -1834,6 +1834,7 @@ private boolean isAnchor(int index) {
         sl.dispose();
     }
 
+    @Ignore // there is no guarantee that there will be 8 selected items (can be 7 as well)
```

Comment:
Ideally we would have an open bug ID for an `@Ignore`d test. In this case, it might or might not be worth filing one to fix the test.

Reply:
I filed https://bugs.openjdk.org/browse/JDK-8289909 for this.